pokecli.py

```diff
@@ -195,6 +195,21 @@ def init_config():
     config.action_wait_max = load.get('action_wait_max', 4)
     config.action_wait_min = load.get('action_wait_min', 1)
 
+    if 'pokestop_cooldown' in load:
+        config.pokestop_cooldown = load['pokestop_cooldown']
+    else:
+        config.pokestop_cooldown = 300
+    
+    if 'cooldown_min' in load:
+        config.cooldown_min = load['cooldown_min']
+    else:
+        config.cooldown_min = 300
+
+    if 'cooldown_max' in load:
+        config.cooldown_max = load['cooldown_max']
+    else:
+        config.cooldown_max = 900
+
```

Comment:
Please, use this pattern: `load.get('parameter_you_want', 'default_parameter_value')`